spec/integration/manifest.spec.yml

```diff
@@ -240,6 +240,13 @@ spec:
           description: GitHub repository name of package maintainer.
           type: string
           pattern: '^(([a-zA-Z0-9-]+)|([a-zA-Z0-9-]+\/[a-zA-Z0-9-]+))$'
+        type:
+          description: Describes who owns the package and the level of support that is provided.
```

Comment:
We should probably define what to expect when this value is not set.

In principle I see two options:
* `elastic` as default, as most existing packages are ours. This may give problems if publishers forget to set the value in the future.
* Add this field starting on next package-spec version (2.10.0). Assume `community` as default from this version of the spec, and `elastic` for current packages. We could add something in our CI to ensure that we set the field.

Reply:
++ to defining a default.

I prefer the second option of having `elastic` needing set explicitly for future packages we release.

Reply:
I definitely agree on setting a default and also agree with defaulting to `community` for the next package-spec version. For existing packages, I think I saw a `before` constraint that we can set to satisfy this:

> `elastic` for current packages.

I saw this, for example:

```yaml
  - before: 2.3.0
    patch:
      - op: add
        path: "/properties/release"
        value:
          description: The stability of the package (deprecated, use prerelease tags in the version).
          deprecated: true # See https://github.com/elastic/package-spec/issues/225
          type: string
          enum:
          - experimental
          - beta
          - ga
          default: ga
          examples:
          - experimental
```

Would I do something similar for `owner.type` and default it to `elastic` when the version is before `2.10.0`?

Reply:
Basically, this:
```diff
 # JSON patches for newer versions should be placed on top
 versions:
   - before: 2.10.0
     patch:
       - op: remove
         path: "/definitions/conditions/properties/elastic/properties/capabilities"
+      - op: add
+        path: "/definitions/owner/properties/type"
+        value:
+          description: Describes who owns the package and the level of support that is provided.
+          type: string
+          default: elastic
+          enum:
+            - elastic
+            - partner
+            - community
+          examples:
+            - community
   - before: 2.3.0
```

Reply:
Well, you could replace only the default, with something like:
```
+      - op: replace
+        path: "/definitions/owner/properties/type/default"
+        value: elastic
```

Or directly remove the definition so it cannot be used in older versions to avoid confusion:
```
+      - op: remove
+        path: "/definitions/owner/properties/type"
```

Reply:
I'm leaning towards the "replacing the default" option so that way a value is always present, rather than the definition being absent in some cases. If you don't think that'd be a concern, then I'm okay with removing the definition as well. Either change seems pretty straightforward to add.